cacheops/invalidation.py

```diff
@@ -16,14 +17,20 @@
 __all__ = ('invalidate_obj', 'invalidate_model', 'invalidate_all', 'no_invalidation')
 
 
+@memoize
+def redis_can_unlink():
+    redis_version = redis_client.info()['redis_version']
+    return StrictVersion(redis_version) >= StrictVersion('4.0')
+
+
 @queue_when_in_transaction
 @handle_connection_failure
 def invalidate_dict(model, obj_dict, using=DEFAULT_DB_ALIAS):
     if no_invalidation.active or not settings.CACHEOPS_ENABLED:
         return
     model = model._meta.concrete_model
     prefix = get_prefix(_cond_dnfs=[(model._meta.db_table, list(obj_dict.items()))], dbs=[using])
-    load_script('invalidate')(keys=[prefix], args=[
+    load_script('invalidate', strip=redis_can_unlink())(keys=[prefix], args=[
```

Comment:
So what this does is pass a flag to `load_script` that invokes a "stripping" function that cacheops already implemented. This will remove from the script any lines between `TOSTRIP` tags. Cheesy, but simple.

Reply:
Note that the fn is memoized, so it isn't really getting and checking the version each time.